src/com/carolinarollergirls/scoreboard/core/impl/ScoreBoardImpl.java

```diff
@@ -83,13 +83,21 @@ protected void valueChanged(PermanentProperty prop, Object value, Object last, F
                 lc.setMaximumTime(getRulesets().getLong(Rule.LINEUP_DURATION));
             }
         } else if (prop == Value.UPCOMING_JAM) {
-            removeAll(Period.NChild.JAM);
```

Comment:
This line was required because on restore from autosave there could be multiple jams, leading to problems. I don't see any code addressing that elsewhere in the PR.

Reply:
How can that happen? This is deleting jams incorrectly currently.